docs/BINDING.md

```diff
@@ -6,11 +6,15 @@ For a Tx module and a receiver to be able to connect to each other, some paramet
 
 The binding procedure is as follows:
 
-1. On the Tx side you have three options: (i) press the bind button (for ca. 4 seconds) on the Tx module, (ii) initiate the binding via a CLI command, or (iii) initiate the binding via the mLRS configuration Lua script.
+1. On the Tx side you have three options: (i) press the bind button (for ca. 4 seconds) on the Tx module, (ii) initiate the binding via a CLI command, or (iii) initiate the binding via the mLRS configuration Lua script. If available, the binding can also be initiated via the OLED display.
```

Comment:
Should the options be a nested list?

Reply:
I'm ok with as it is. You sggest I should make it a nested list?

Reply:
Okay, fine as is then.  :)